_i18n/en.yml

```diff
@@ -0,0 +1,81 @@
+---
+global:
+  brazilian portuguese: Português
+  english: English
+header:
+  stories: Stories
+  explore: Explore
+  about: About
+support:
+  fund_us: Support the project
+home:
+  about: Artificial Intelligence for social control of public administration
+  about_rosie: Operação Serenata de Amor created a Robot - An artificial intelligence capable of analysing each request for reimbursement from congresspeople and identify it’s illegal probability.
+  last_histories: Last published histories
+  loading: Loading...
+  more_posts: More posts
+footer:
+  about: about
+  github: Serenata on github
+  contact: contact
+  partners: partners
+stories:
+  published: Published stories
+  loading: Loading...
+explore:
+  reports: Reports
+  about_reports: Rosie, our Robot, in numbers
+  tweet_about: Tweet about this number
+  chamber_reports: Reports done to the Chamber of Deputies
+  suspitions: Different congresspeople reported for suspicious reimbursements
+  questionated: thousand Brazilian Reals in suspicious expenses questioned
+  paragraph_one: Those values correspond to 371% of what we raised with our crowdfunding campaign (~U$ 25.000,00)
+  paragraph_two: Want to know how the report system works?
+  more: You can read more in this post
+about:
+  title: Serenata de amor operation.
+  who_contributed: People who contributed
+  paragraph_one: We are a project who make use of artificial intelligence to audit public accounts and fight corruption. The idea came from the data scientist
+  paragraph_two:  when he realized about the many possibilities to use technology to inspect congressperson’s spends. When he shared the idea with friends, the project growth in strength in no time and the team was formed with the goal to finish the project.
```

Comment:
Some minor edits:

_when he started to wonder about many possibilities to use technology to inspect congresspeople expenses. When he shared this idea with friends, the project growth in strength in no time and the team was gathered with the goal of making this project happen._